examples/undocumented/python/structure_multiclass_bmrm.py

```diff
@@ -68,7 +68,7 @@ def structure_multiclass_bmrm(fm_train_real=traindat,label_train_multiclass=labe
 	ppbmrm_out = sosvm.apply()
 	count = 0
 	for i in range(ppbmrm_out.get_num_labels()):
-		yi_pred = ppbmrm_out.get_real_vector("labels_vector")[i]
+		yi_pred = ppbmrm_out.get("labels_vector")[i]
```

Comment:
very nice that you are fixing those things on the fly as well.
It might be worth grepping for those in the ipython notebooks as well